PyInstaller/utils/tests.py

```diff
@@ -41,7 +41,7 @@ def importorskip(*modules):
     for m in modules:
         try:
             __import__(m)
-        except ImportError:
+        except Exception:
```

Comment:
I've re-thought this: `Exception` is to wide, as it would catch anything: I suggest using
`except (ImportError, SyntaxError):`